app/components/contributor-list/component.js

```diff
@@ -1,21 +1,46 @@
 import { filterBy } from '@ember/object/computed';
-import { computed } from '@ember/object';
 import Component from '@ember/component';
-import { loadRelation } from 'ember-osf/utils/load-relationship';
+import { inject as service } from '@ember/service';
+
+import { task } from 'ember-concurrency';
+
+
+export default Component.extend({
+    store: service(),
 
-const ContributorListComponent = Component.extend({
     tagName: 'ul',
     node: null,
+    page: 1,
 
     bibliographicContributors: filterBy('contributors', 'bibliographic', true),
 
-    contributors: computed('node', function() {
-        return loadRelation(this.get('node'), 'contributors');
+    init() {
+        this._super(...arguments);
+        this.set('contributorsList', this.get('contributors') || []);
```

Comment:
Stuff that depends on passed-in attributes should be in `didReceiveAttrs`, not `init`